relayer/relay.go

```diff
@@ -259,7 +260,7 @@ func (r *Relayer) processMessages(ctx context.Context) {
 			}
 
 			// if message is received we can remove the message from db
-			if messageReceived {
+			if message.EventType == events.EmitMessage && messageReceived {
```

Comment:
please synchronize the implementation of MessageReceived similar to other chains so that we don;t have to add eventType check here